src/cmd/create.go

```diff
@@ -181,6 +182,15 @@ func createContainer(container, image, release string, showCommandToEnter bool)
 		return nil
 	}
 
+	s := spinner.New(spinner.CharSets[9], 500*time.Millisecond)
+
+	if logLevel := logrus.GetLevel(); logLevel < logrus.DebugLevel && isatty.IsTerminal(1) {
```

Comment:
I am curious. Why are we moving the entire chunk of code that creates the spinner, instead of just adding the extra condition to the existing chunk in-place?